challenges-completed.js

```diff
@@ -10,9 +10,18 @@ document.addEventListener('DOMContentLoaded', function (event) {
     updateIndex('./data.json')
   })
 
+  ipc.on('confirm-clear-response', function (response) {
+    if (response === 1) return
+    else clearAllChallenges()
+  })
+
   var clearAllButton = document.getElementById('clear-all-challenges')
 
   clearAllButton.addEventListener('click', function (event) {
```

Comment:
Actually I've fixed it in the next stuff I'm working on since I'm refactoring some things out to store the user data properly :+1: